source/docs/application-notes/manual-pid-tuning.rst

```diff
@@ -0,0 +1,334 @@
+Arm Tuning with TorqueCurrentFOC
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Tuning an Arm is very similar to tuning a turret, just with the addition of needing to account for gravity. As such, the process is nearly identical, except for a small section dedicated to dialing in the kG term.
+
+The steps:
+ 1. Zero all PID gains.
+ 2. Increase kG and find the smallest possible kG that stops the arm from moving.
+ 3. Increase kG and find the largest possible kG that stops the arm from moving.
+ 4. Set kG to the middle of the two.
+ 5. Set a setpoint relatively nearby (typically 0.1 mechanism rotations).
+ 6. Increase kS until the arm starts moving, then back off to just before that movement.
+ 7. Increase kP until you notice significant overshoot.
+ 8. Increase kD until the overshoot stops happening.
+ 9. Repeat steps 7 and 8 until increasing kD results in more oscillation, or until the system oscillates on its way to the setpoint. If oscillation on the way to setpoint is seen, decrease kD until it stops. If overshoot in general is happening and kD is already at max, reduce kP until it stops.
+ 10. Verify gains work for other setpoints as well. Tune kP/kD as appropriate for most general cases.
+
+.. dropdown:: "Why" for each step
+
+   1. We start with PID gains at 0 to isolate as many of the forces in play as possible, and iteratively get closer to the "ideal" gains.
+   2. The kG gain is meant to counteract the force of gravity, however the force of friction is also at play in an arm. The lowest possible kG that prevents the system from moving is the lower bound of the gravity and friction component.
+   3. The highest possible kG that prevents the system from moving is the upper bound of the gravity and friction component.
+   4. Setting kG to the middle point of the lower and upper bounds is a good approximation for the true effect of gravity, removing the force of friction.
+   5. A nearby setpoint ensures the system response should be relatively small to start with when tuning.
+   6. The kS gain is meant to reduce the effect of friction, so the largest possible value that still prevents the system from moving will reduce the effect of friction in general.
+   7. The kP gain will control how quickly the system gets to the setpoint, however in TorqueCurrentFOC modes there is no natural dampening force, so overshoot is expected at the beginning. Once that happens kD should be tuned.
+   8. The kD gain will effectively slow down the system as it reaches the setpoint, increasing it will increase the force slowing it down, so it should be increased until the system no longer overshoots.
+   9. In general, the system wants as high a kP gain as possible to decrease the time taken to get to the setpoint. This also requires a high kD gain to properly dampen the system. The limit of how high the kP/kD term can be is determined by the system latency, at which point the oscillation is impossible to avoid. The goal of repeating steps 7 and 8 is to find that limit.
```

Comment:
I think this explanation that the goal is to have as high a kP gain as possible, limited by oscillation, is very important. Perhaps this should be stated above outside of the mechanism specific sections or at least not "hidden" in the "why" section.